src/CloudNative.CloudEvents/CloudEventsSpecVersion.cs

```diff
@@ -1,15 +1,146 @@
-﻿// Copyright (c) Cloud Native Foundation. 
+﻿// Copyright 2021 Cloud Native Foundation. 
 // Licensed under the Apache 2.0 license.
 // See LICENSE file in the project root for full license information.
 
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Mime;
+using static CloudNative.CloudEvents.CloudEventAttribute;
+
 namespace CloudNative.CloudEvents
 {
-    public enum CloudEventsSpecVersion
+    // TODO: Rename to CloudEventSpecVersion? Using the plural form feels a little odd.
```

Comment:
Agreed

Reply:
Cool. I'll do that that in a follow-up PR, in a single commit, rather than doing it now - otherwise it will disrupt our review discussion. Have made a note for myself.

(Annoyingly, I can see how the plural form is more formally correct here. I hate it when we have to decide between "formally correct" or "feels better"...)